Update build.py

alexandercyu opened the pull request:
Removed build_tutorials code
Resolves #4

alexandercyu committed:
Update build.py

Removed build_tutorials code

mishaschwartz reviewed (changes requested):
- the subprocess import can also be removed
- the `TUTORIALS_PATH` can also be removed
- there are too many spaces before `if __name__ == "__main__"`

These are all things that your IDE should be highlighting for you. If you're not seeing those warnings, let me know and we can discuss how to configure your IDE properly.

alexandercyu commented:
- removed subprocess import
- removed TUTORIALS_PATH
- Just noticed the tip at the top of the IDE that the Python interpreter wassn't installed.  I think because it used the lightbulb logo I just dismissed it as the tip of the day.  Turned it on now.

alexandercyu committed:
Update build.py

- Removed subprocess import
- removed TUTORIAL_PATH
- removed spacing
